tests/Unit/Domain/Creators/Test_BinaryCompactObject.cpp

```diff
@@ -451,6 +454,7 @@ void test_bns_domain_with_cubes() {
 
   // Outer shell:
   constexpr double outer_radius = 32.4;
+  constexpr double opening_angle = 90.0;
```

Comment:
Same comment as above. You'll probably need to edit the `create_option_string` as well to take arbitrary values.